### PR TITLE
[TD] Name uniformity for TechDraw_Tree Icons

### PR DESCRIPTION
**Pull request by bitacovir**
Many TechDraw icons don't follow name conventions and are not SVG plain format. Help to fix this was required.

This commit replaces a group of SVG files (Tree_*) with plain SVG format and changes their names following FC convention.  Also, it makes the necessary changes on TechDraw.qrc files, and several cpp files.

Forum Discussion: https://forum.freecadweb.org/viewtopic.php?f=34&t=58401

### CODE REVIEW

**Review by luzpaz — APPROVED** [review]

File: src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreeHatch.svg [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        </dc:creator>
+        <dc:title>TechDraw_Tree_Hatch</dc:title>
+        <dc:date>2016-04-27</dc:date>
+        <dc:relation>http://www.freecadweb.org/wiki/index.php?title=Artwork</dc:relation>
+        <dc:publisher>
+          <cc:Agent>
+            <dc:title>FreeCAD</dc:title>
+          </cc:Agent>
+        </dc:publisher>
+        <dc:identifier>FreeCAD/src/Mod/TechDraw/Gui/Resources/icons/TechDraw_Tree_Hatch.svg</dc:identifier>
```
Comment (luzpaz) [inline src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreeHatch.svg:37]: rename

File: src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreePage.svg [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        </dc:creator>
+        <dc:title>TechDraw_TreePage</dc:title>
+        <dc:date>2016-04-27</dc:date>
+        <dc:relation>http://www.freecadweb.org/wiki/index.php?title=Artwork</dc:relation>
+        <dc:publisher>
+          <cc:Agent>
+            <dc:title>FreeCAD</dc:title>
+          </cc:Agent>
+        </dc:publisher>
+        <dc:identifier>FreeCAD/src/Mod/TechDraw/Gui/Resources/icons/TechDraw_Tree_Page.svg</dc:identifier>
```
Comment (luzpaz) [inline src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreePage.svg:36]: rename

File: src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreeMulti.svg [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title>TechDraw_TreeMulti</dc:title>
+        <dc:creator>
+          <cc:Agent>
+            <dc:title>[WandererFan]</dc:title>
+          </cc:Agent>
+        </dc:creator>
+        <dc:title>TechDraw_Tree_Multi</dc:title>
```
Comment (luzpaz) [inline src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreeMulti.svg:67]: rename

**Review by luzpaz — CHANGES_REQUESTED** [review]
Pending merge from https://github.com/bitacovir/FreeCAD/pull/1

File: src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreeMulti.svg [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        <dc:date>2016-10-29</dc:date>
+        <dc:relation>http://www.freecadweb.org/wiki/index.php?title=Artwork</dc:relation>
+        <dc:publisher>
+          <cc:Agent>
+            <dc:title>FreeCAD</dc:title>
+          </cc:Agent>
+        </dc:publisher>
+        <dc:identifier>FreeCAD/src/Mod/TechDraw/Gui/Resources/icons/TechDraw_Tree_Multi.svg</dc:identifier>
```
[the remark this answers is not in the document]
Comment (bitacovir) [reply src/Mod/TechDraw/Gui/Resources/icons/TechDraw_TreeMulti.svg]: All 11 done :)

**Comment by bitacovir** [comment]
@luzpaz All svg files checked and updated.

**Review by luzpaz — APPROVED** [review]
LGTM

**Pull request merged**